config/index.js

```diff
@@ -33,10 +33,15 @@ const kubeClient = new kube.Client({ backend: kubeBackend })
 
 const logger = pino({
   serializers: {
-    err: pino.stdSerializers.err
+    err: pino.stdSerializers.err,
+    messageKey: 'message'
```

Comment:
potentially breaking as current default for pino is `msg`

Reply:
Yeah, I just fixed that.